vagrant/global_vars.yml

```diff
@@ -12,9 +12,9 @@ install_pkgs:
 - iptables-utils
 - iptables-services
 - docker
-- kubeadm
-- kubelet
-- kubectl
+- kubeadm-1.7.8
+- kubelet-1.7.8
+- kubectl-1.7.8
```

Comment:
What happens if you don't specify the version explicitly here? Would the latest RPMs get pulled? If so, is that bad?

I guess my main thought is that the "latest" RPMs will always be there, whereas I don't know what the retention policy of older RPMs is. Not sure if breaking because of a version change (like we just did) or breaking because the RPMs are gone are really any different, though. :)

Reply:
Yes, you end up with the latest RPMs - which are as I write this 1.8.1. This version changes some behaviors and command output in a couple of places and this breaks the current playbooks.
As of this moment the quickstart guide can't run to completion because of that.

I'd like to follow up on this change later with some that make it usable with1.8.1 and possibly use the version variable in global_vars.yml to determine what version of the packages are used and conditionalize behaviors.
However, in the short term I hoped to keep the scope small.

Reply:
I say we can let the scope creep up just a little bit: replace the other instances of 1.7.8 with the version variable. :)